pig/src/test/java/org/jboss/pnc/bacon/pig/impl/utils/HashUtilsTest.java

```diff
@@ -59,4 +59,13 @@ void shouldGiveDifferentHashForAddedFile() {
         assertThat(originalHash).isNotEqualTo(exactCopyHash);
     }
 
+    @Test
+    void shouldIgnoreSomePaths() {
+
+        String originalHash = HashUtils.hashDirectory(testDirs.resolve("original"));
+        String copyIgnoreFileHash = HashUtils.hashDirectory(testDirs.resolve("copy-with-file-to-be-ignored"), path -> {
```

Comment:
how does this test check if the file was ignored?

Reply:
I'll double check. Maybe I made a booboo again :(

Reply:
Oh the sha should be different if both were different. Of course I forgot to add the assert line 🤦‍♂️

Reply:
https://github.com/project-ncl/bacon/pull/558 to fix this. Thanks for pointing it out! :)